J-Runner/MainForm.cs

```
@@ -405,17 +405,17 @@ private void saveToLog(bool bFormClosing)
                     // won't be able to close. Append today's date
                     // to the file path and reattempt the write.
                     Console.WriteLine("Couldn't write console log to " + file);
-                    Console.WriteLine(e.ToString() + " " + e.Message);
+                    Console.WriteLine(e.GetType().ToString() + " " + e.Message);
 
                     try
                     {
-                        file += DateTime.Now.ToString("yyyyMMdd");
+                        file += "." + DateTime.Now.ToString("yyyyMMdd");
```

Why?

---

Just a choice of file name, appending the date seemed reasonable. The exact file name in particular doesn't matter, just that it is different than the main log file.

Ran in to some weird behaviour on my own system related to the log file (see the screenshot) that was preventing the log from being saved.

Alternatives would be to throw up a file save dialog and force one to choose a new location, or just ignoring the error and discarding the log (though I'd rather have the opportunity to save it)

---

OK, I don't love a dot because *sometimes* weird extension things happen. would you be ok with a dash or underscore? That would be my preference.

---

No preference from my side, whatever you prefer is fine. I can make the change later today.

---

@Octal450 posted an update to keep the extension the same and change the root of the filename.